kwok/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start from a base image
+FROM registry.access.redhat.com/ubi8:latest
```

Comment:
and we use an existing kwok image as base?

Reply:
could you add the kwok base image here?

Reply:
You are telling me you couldn't find such an image?

Reply:
we might need Kwok base image in Quay. since docker images have a rate limit. we might have to find something that resides with our goal.

Reply:
not sure we'll build frequently enough to hit that rate limit

Reply:
Done